html2asketch/nodeToSketchLayers.js

```diff
@@ -62,6 +62,14 @@ function isSVGDescendant(node) {
   return (node instanceof SVGElement) && node.matches('svg *');
 }
 
+function parseFontWeight(fontWeight) {
+  // Support 'bold' and 'normal' for Electron compatibility.
+  if (fontWeight === 'bold' || fontWeight === 'normal') {
+    return fontWeight;
```

Comment:
how about 

```
if (fontWeight === 'bold') {
    return 700;
} else if (fontWeight === 'normal') {
   return 400;
}
```

will that work for your electron app? It feels a bit more solid to always return a number from this function.